HeterogeneousCore/SonicTriton/interface/TritonData.h

```diff
@@ -40,6 +43,16 @@ class TritonData {
   bool setShape(const ShapeType& newShape) { return setShape(newShape, true); }
   bool setShape(unsigned loc, int64_t val) { return setShape(loc, val, true); }
 
+  void setConverterParams(const edm::ParameterSet& conf) {
+    converterName_ = conf.getParameter<std::string>("converterName");
+  }
+  template <typename DT>
+  void createConverter() const { 
```

Comment:
this function (and the above `setConverterParams()` should go in the `private` section of this class